src/openzaak/config/templates/config/config_detail.html

```diff
@@ -9,31 +9,64 @@ <h1 class="display-4">Service configuratie</h1>
 <div class="container text-left">
 
     <div class="text-right">
-        <a href="{% url 'config-change' %}" class="btn btn-sm btn-secondary">Change</a>
+        <a href="{% url 'config-nlx' %}" class="btn btn-sm btn-secondary">Change</a>
     </div>
+    <!-- NLX config -->
+    <section>
+        <div class="text-center">
+            <h3>NLX</h3>
+        </div>
 
-    <div class="text-center">
-        <p class="lead">NLX</p>
-    </div>
+        <table class="table table-striped">
+            <thead>
+                <tr>
+                    <th scope="col">Parameter</th>
+                    <th scope="col">Waarde</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td><strong>NLX directory</strong></td>
+                    <td>{{ nlx.directory_url| urlize }}</td>
+                </tr>
+                <tr>
+                    <td><strong>Outway address</strong></td>
+                    <td>{{ nlx.outway| urlize }}</td>
+                </tr>
+            </tbody>
+        </table>
+    </section>
+
+    <!-- Internal API config -->
+    <section>
+        <div class="text-center">
+            <h3>Open Zaak services</h3>
+        </div>
+
+        <table class="table table-striped">
+            <thead>
+                <tr>
+                    <th scope="col">Service</th>
+                    <th scope="col" class="text-center">Enabled</th>
```

Comment:
This needs some sort of info icon with a popover with additional information explaining the meaning of this setting.

Reply:
tooltip is added